web/src/ts/routes/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react'
+import { Redirect, Route, Switch } from 'react-router-dom'
+
+import HomePageContainer from '../pages/HomePageContainer/HomePageContainer'
+
+export const Routes: FC = () => {
+  return (
+    <Switch>
+      <Route exact path='/' render={ () => <HomePageContainer /> } />
```

Comment:
You can ommit unneccesary code like () => 
```suggestion
      <Route exact path='/' component={HomePageContainer} />
```